src/test/ui/consts/ptr_is_null.rs

```diff
@@ -0,0 +1,25 @@
+// compile-flags: --crate-type=lib
+
+#![feature(const_ptr_is_null, const_panic)]
+
+const FOO: &usize = &42;
+
+pub const FOO_PTR_IS_NULL: bool = (FOO as *const usize).is_null();
+pub const CHECK: () = assert!(FOO_PTR_IS_NULL);
+//~^ ERROR any use of this value will cause an error
```

Comment:
That's such a weird test, the assertions are the opposite of what we want!
Please either add a comment, or (better IMO) make it a normal positive assertion.